test/acceptance/tests.py

```diff
@@ -500,10 +500,9 @@ def test_staff_area_panel(self, panel_name, button_label):
         # Click on the button and verify that the panel has opened
         self.staff_area_page.click_staff_toolbar_button(panel_name)
         self.assertEqual(self.staff_area_page.selected_button_names, [button_label])
-        self.assertIn(
-            u'openassessment__{button_name}'.format(button_name=panel_name),
-            self.staff_area_page.visible_staff_panels[0]
-        )
+        visible_panels = self.staff_area_page.visible_staff_panels
```

Comment:
Note to reviewers: no change in functionality, just checking length of visible panels.